<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleSearcher</class>
 <widget class="QWidget" name="SimpleSearcher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimpleSearcher</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="filterFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>1</number>
      </property>
      <item row="0" column="4" rowspan="2">
       <widget class="QToolButton" name="findButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Find</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/images/find</normaloff>:/images/find</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelLimit">
        <property name="text">
         <string>Limit:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="limitComboBox">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>50</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelType">
        <property name="text">
         <string>Type:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelName">
        <property name="text">
         <string>Name:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="searchLineEdit"/>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QPushButton" name="addTypeButton">
        <property name="text">
         <string>Add type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="typeComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="viewFrame">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>1</number>
      </property>
      <item row="0" column="1">
       <widget class="QToolButton" name="nextPageButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/images/forward</normaloff>:/images/forward</iconset>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QToolButton" name="prevPageButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/images/backward</normaloff>:/images/backward</iconset>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="6">
       <widget class="QWebView" name="webView">
        <property name="url">
         <url>
          <string>about:blank</string>
         </url>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelResults">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Results:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" colspan="3">
       <widget class="QComboBox" name="resultComboBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
